releases/release-1.21/README.md

```diff
@@ -60,6 +60,7 @@ The 1.21 release cycle is proposed as follows:
 | **Begin [Burndown]** (MWF meetings) | Lead | Mon March 1 | week 8 | [1.21-blocking], [master-blocking], [master-informing] |
 | **Call for [Exceptions][Exception]** | Lead | Mon March 1 | week 8 | |
 | Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon March 1 | week 8 | |
+| **Begin Feature blog freeze** | Comms Lead | Mon March 1 | week 8 | |
```

Comment:
We are actively trying to not keep milestones on Mondays/Fridays. Can we shift the feature blog freeze to Tuesday March 2?

cc: @divya-mohan0209

Reply:
@palnabarun I completely agree with your point, however the Communication for the freeze has already gone out in the email sent by @annajung last week. I suggest we retain just this one because there's a huge chance we might create confusion instead of simplifying things. However, I'm open to discussion if there's a better way to make this happen.

Reply:
Ah okay. Good point. In light of that let's keep the March 1 deadline.